packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/presentation-components",
-  "version": "5.9.0",
+  "version": "6.0.0",
```

Comment:
TODO: manually change to `5.10`. This doesn't have to be a major, but `changesets` thinks it should.

Reply:
Updated, @saskliutas please review